scripts/examples/proposeAddCollateralizable.ts

```diff
@@ -12,33 +12,39 @@ import {Contract} from "ethers";
 import {getCollateralVaultAddress} from "../../common/contracts";
 import {verifyVotingPower} from "./util";
 
-const collateralizableAddress = process.env.COLLATERALIZABLE_ADDRESS
+const collateralizableAddresses = process.env.COLLATERALIZABLE_ADDRESSES
 
 const getAddCollateralizableProposalCall = async (): Promise<ProposalCall> => {
   const vault: Contract = await ethers.getContractAt('CollateralVault', getCollateralVaultAddress())
 
-  const approvals = [
-    {
-      collateralizableAddress: collateralizableAddress,
+  const parsedAddresses = (collateralizableAddresses || '').split(',').map(x => x.trim()).filter(x => x !== '')
+  if (!parsedAddresses.length || parsedAddresses.filter(x => !isValidEthereumAddress(x)).length > 0) {
+    console.error(`ERROR: COLLATERALIZABLE_ADDRESSES variable is not populated or does not contain a valid comma-separated list of addresses.`)
+    process.exit(1)
+  }
+
+  const approvals = parsedAddresses.map(collateralizableAddress => {
+    return {
+      collateralizableAddress,
       isApproved: true
-    },
-  ]
+    }
+  })
 
   const approvalUpdates = await vault.upsertCollateralizableContractApprovals.populateTransaction(approvals)
 
   return newProposalCall(await vault.getAddress(), approvalUpdates.data!)
 }
 
 /**
- * Usage: COLLATERALIZABLE_ADDRESS="someAddressHere" npx hardhat run --network localhost scripts/examples/proposeAddCollateralizable.ts
+ * Usage: COLLATERALIZABLE_ADDRESSES="someAddressHere" npx hardhat run --network localhost scripts/examples/proposeAddCollateralizable.ts
  */
 async function main() {
   // Just to prompt for address now rather than later
   const signer = await getSigner(ethers)
 
   await verifyVotingPower(ethers, signer)
 
-  const description = `Adds ${collateralizableAddress} as an approved collateralizable contract in the vault ${getCollateralVaultAddress()}`
+  const description = `Adds ${collateralizableAddresses} as an approved collateralizable contract in the vault ${getCollateralVaultAddress()}`
```

Comment:
Only feedback would be this grammar is off now.
```suggestion
  const description = `Adds ${collateralizableAddresses} as approved collateralizable contracts in the vault ${getCollateralVaultAddress()}`
```